aws-gateway-controller.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: aws-gateway-controller
```

Comment:
the name of the repo is different which is https://github.com/aws/aws-application-networking-k8s/ but I chose this name because the container image was named `public.ecr.aws/aws-application-networking-k8s/aws-gateway-controller`

Reply:
Let's rename the package name to `aws-application-networking-k8s` for consistency and add `aws-gateway-controller=${{package.full-version}}` to `provides:`.